app/src/main/res/layout/subscription_item.xml

```diff
@@ -61,4 +87,15 @@
 
     </FrameLayout>
 
-</RelativeLayout>
+    <TextView
+        android:id="@+id/fallbackTitleLabel"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:ellipsize="end"
+        android:gravity="center"
+        android:padding="6dp"
+        android:textColor="?android:attr/textColorPrimary"
+        android:background="@drawable/bg_round"
```

Comment:
Something went wrong here. This is the main title, not the fallback title. It should not have a round background.

Reply:
This is indeed the fallback title, thus has a round background